pkgs/development/misc/rappel/default.nix

```diff
@@ -0,0 +1,34 @@
+{ clang
+, fetchFromGitHub
+, libedit
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  name = "rappel";
+
+  src = fetchFromGitHub {
+    owner = "yrp604";
+    repo = "rappel";
+    rev = "95a776f850cf6a7c21923a2100b605408ef038de";
+    sha256 = "0fmd15xa6hswh3x48av4g1sf6rncbiinbj7gbw1ffvqsbcfnsgcr";
+  };
+
+  buildInputs = [ libedit ];
+  nativeBuildInputs = [ clang ];
+
+  makeFlags = "CC=clang";
+
+  installPhase = ''
+    make
+    cp -r bin $out/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/yrp604/rappel";
+    description = "A pretty janky assembly REPL";
+    license = "stdenv.lib.licenses.free";
```

Comment:
https://github.com/yrp604/rappel/blob/master/LICENSE looks like a 4-clause BSD license, so I think this should be

```
license = licenses.bsdOriginal;
```

(not a string)